README.md

```diff
@@ -12,6 +12,11 @@ module that contains logic which is able to translate this into a form that is s
 for quantum algorithms. The conversion first creates a FermionicOperator which must then be mapped,
 e.g. by a Jordan Wigner mapping, to a qubit operator in readiness for the quantum computation.
 
+TODO: Update the above:
+- do not mention FermionicOperator but rather general second-quantized ops (at least until we have first-quantized ones)
+- introduce problem classes
+- we don't have any physics yet but the above could be written such that this is not all about chemistry
+
```

Comment:
This TODO does not see relevant any more. You removed all FermionicOperator mentions from what I see and introduced in the code snipper the problem classes.

Reply:
This is not true. See e.g. line 12:

> The conversion first creates a FermionicOperator